setup.py

```diff
@@ -327,7 +327,12 @@ def run(self):
 tests_require.append('nbval')
 tests_require.append('tabulate')
 
-extras_require['mypy'] = ['mypy==0.782', 'types-protobuf==3.18.4']
+extras_require['lint'] = [
+    'clang-format==13.0.0',
+    'flake8',
+    'mypy==0.782',
+    'types-protobuf==3.18.4',
+]
```

Comment:
As you mentioned in the PR description, I think adding a new command like "python setup.py lint" and removing existing typecheck should be a good idea.

Reply:
Thanks for the comments!
For `python setup.py lint` command, I summarized my opinion in https://github.com/onnx/onnx/pull/3988#issuecomment-1031139310